scripts/storage-emulator-integration/tests.ts

```diff
@@ -263,9 +263,51 @@ describe("Storage emulator", () => {
           });
         });
 
-        // TODO(abehaskins): This test is temporarily disabled due to a credentials issue
-        it.skip("should handle large (resumable) uploads", async () => {
-          await testBucket.upload(largeFilePath, { resumable: true });
+        it.only("should handle large (resumable) uploads", async () => {
+          const fileName = "test_upload.jpg";
+          const uploadURL = await supertest(STORAGE_EMULATOR_HOST)
+            .post(`/upload/storage/v1/b/${storageBucket}/o?name=${fileName}&uploadType=resumable`)
+            .send({})
+            .set({
+              Authorization: "Bearer owner",
+            })
```

Comment:
Nit: one line

Reply:
Done

Reply:
Actually prettier complains if it's not 2 separate lines, reverting to what it was originally